login/DataModels.js

```diff
@@ -0,0 +1,61 @@
+const sq =  require("./DataConnection.js");
+const sequelize = sq.sequelize;
+const Sequelize = sq.Sequelize;
+const Roles = sequelize.define("roles", {
+    id:{type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+        allowNull: false
+    },
+    role_name:{
+        type: Sequelize.STRING,
+        allowNull: false
+    }
+});
+
+const User = sequelize.define("users", {
```

Comment:
I'd suggest splitting this up even further, create a folder called "Models" and define each model in a separate file to adhere to Open Closed principle.

Reply:
Yes,its good idea,thanks